client/src/main/scala/com/all4journey/webapp/util/Fragments.scala

```diff
@@ -0,0 +1,151 @@
+package com.all4journey.webapp.util
+
+import com.wix.accord._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/rjkernick/all4journey/file/2503581573/issues/source?bid=3246460&fileBranchId=3273764#l3)